data.sql

```diff
@@ -13,3 +13,61 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (10, 'Blossom', '1998-09-13', 3, true, 17);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (11, 'Ditto', '2022-05-14', 4, true, 22);
 
+/* Insert the following data into the owners table*/
+insert into owners (full_name, age)
+values ('Sam Smith', 34);
+
+insert into owners (full_name, age)
+values ('Jennifer Orwell', 19);
+
```

Comment:
Great job @Abdessittir 🤝

- [ ] You may also add this data to the `owners` table. You seem to have missed it.
![bob](https://user-images.githubusercontent.com/63917725/213413852-e2885830-6df2-4f79-a9e0-646e2da838c0.PNG)